packages/watchguard_firebox/data_stream/log/elasticsearch/ingest_pipeline/pipeline_traffic.yml

```
@@ -1098,11 +1094,13 @@ processors:
             field: error.message
             value: 'Processor {{{_ingest.on_failure_processor_type}}} with tag {{{_ingest.on_failure_processor_tag}}} in pipeline {{{_ingest.on_failure_pipeline}}} failed with message: {{{_ingest.on_failure_message}}}'
   - community_id:
+      tag: src_dst_community_id
       source_ip: watchguard_firebox.log.source_ip
       source_port: watchguard_firebox.log.source_port
       destination_ip: watchguard_firebox.log.destination_ip
       destination_port: watchguard_firebox.log.destination_port
       ignore_missing: true
+      ignore_failure: true
```

Is this because one or more of the ports is zero? Wonder if it would be preferred to use an if statement instead. I'm always a bit skeptical of turning on ignore_failure, since we could hide other legitimate failures. On the other hand, if this is the only known failure mode, perhaps it's okay.

---

It was because of the error when port is 0. I've changed to use `if` instead